Cargo.toml

```diff
@@ -11,6 +11,8 @@ members = [
   "in-vehicle-digital-twin",
   "samples/command/consumer",
   "samples/command/provider",
+  "samples/mixed/consumer",
```

Comment:
[10:56 AM] Ash Beitz
I just had a quick look at your comments.  The "mixed" sample demonstrates the use of command and property intents.  It also shows multiple properties and commands co-existing in one sample.  I wanted to reduce the name for the sample to one word, like the previous samples ("property" and "command").  This makes running the sample's provider and consumer easier.  If you have a suggestion for a better name, then please let me know.

[10:56 AM] Ash Beitz
I was going to use "dog-mode", but the dog mode sample is really part of the Chariott domain, not Ibeji.

[10:59 AM] William Lyles
I think dog mode would still be an appropriate name. the dog mode application may not exist at this level, but the idea is that this is more or less what would be happening at the ibeji level in the dog mode scenario. I'll see if I can come up with alternatives, but I think it would be beneficial to at least document what this sample does in a readme

Reply:
I agree to adding documentation to the readme to explain the purpose of this sample.  I have now done this.  I prefer not to refer to it as the "dog-mode" sample, as Ibeji and Chariott (where the dog-mode demo lives) are standalone projects.   If someone comes up with a better name, then I am happy to change it.